api/firm/firm.go

```diff
@@ -81,6 +84,9 @@ LIMIT 1;`, firm_id)
 				&temp.Balance,
 				&temp.Size,
 				&temp.Execs,
+				&temp.Assocs,
```

Comment:
This would throw an error because you did not specify the number of assocs, COOs, and CFOs in the SQL query above. Modify the query and then I can merge. Also, if you are changing the output of any endpoints, please make sure to update the documentation (README.md)